### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by felixzieger**
This will be merged after https://github.com/meshcloud/meshcloud-docs/pull/529

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # User Guides
 
+NOTE: This repository will be archived. It's content is now part of [meshcloud-docs](https://github.com/meshcloud/meshcloud-docs/).
+
 #### Hi there :sheep:
 Welcome to our user guide templates which will support you in enabling users :crown: navigating and using the multi-cloud landscape you create in your organization. You can customize and tailor the guides to the requirements and needs of your organization. Believe us it is easy :+1: We got the general things covered so you can focus on the final touches :sparkles: .
```
Comment (Prerna21122112) [inline README.md:6]:
```suggestion
Welcome to our user guide templates which will support you in enabling users :crown (what does crown mean here?): navigating and using the multi-cloud landscape you create in your organization. You can customize and tailor the guides to the requirements and needs of your organization. Believe us, it is easy :+1: We got the general things covered so you can focus on the final touches :sparkles: . wat does +1 and sparkle stand for?
```

**Review by Prerna21122112 — APPROVED** [review]

**Review by Prerna21122112 — APPROVED** [review]

**Comment by uleMeshcloud** [comment]
As far as I  remember the idea of the User Guide is to provide our customers an initial information base regarding meshStack, so they can use this and adapt/add their specific information to it. Since a lot of customers provide their end users a customer specific guide anyways. But I guess it had not been used anyways and we don't t offer that anymore? Or what is the reason behind it?

Btw, there also open PR in the USer Guide from Younghi. Shall migrate it to docs too?

**Review by uleMeshcloud — COMMENTED** [review]
As far as I remember the idea of the User Guide is to provide our customers an initial information base regarding meshStack, so they can use this and adapt/add their specific information to it. Since a lot of customers provide their end users a customer specific guide anyways. But I guess it had not been used anyways and we don't t offer that anymore? Or what is the reason behind it?

Btw, there also open PR in the USer Guide from Younghi. Shall migrate it to docs too?

**Comment by felixzieger** [comment]
> As far as I remember the idea of the User Guide is to provide our customers an initial information base regarding meshStack, so they can use this and adapt/add their specific information to it. Since a lot of customers provide their end users a customer specific guide anyways. But I guess it had not been used anyways and we don't t offer that anymore? Or what is the reason behind it?

I believe that https://github.com/meshcloud/User-Guides should go into [meshcloud-docs](https://docs.dev.meshcloud.io/). The source code for both is public. If a customer wants to copy and adapt guides, it is as easily possible from the meshcloud-docs repo as it is from the User-Guides repository.
This would have the advantage of also showing the guides on meshcloud-docs for teams that only consult the public-facing documentation of meshStack.

> Btw, there also open PR in the USer Guide from Younghi. Shall migrate it to docs too?

Ah right, I will also migrate it over. Thanks!

**Pull request merged**